src/main/scala/tests/givenSignatures.scala

```diff
@@ -0,0 +1,40 @@
+package tests
+
+package givenSignatures
+
+
+
+class GivenClass {
+    trait B
+    trait C[T]
+    val r: Int = 5
+    type R = Int
+    given R = r
```

Comment:
This given in the docs links to itself, but instead should link to `type R`.

Reply:
Fixed